homeassistant/components/decora_wifi/light.py

```diff
@@ -114,8 +57,8 @@ def turn_on(self, **kwargs):
         attribs = {"power": "ON"}
 
         if ATTR_BRIGHTNESS in kwargs:
-            min_level = self._switch.data.get("minLevel", 0)
             max_level = self._switch.data.get("maxLevel", 100)
+            min_level = self._switch.data.get("minLevel", 100)
```

Comment:
Is this a bugfix?

Reply:
Reverting it to `min_level = self._switch.data.get("minLevel", 0)` as it is in the current integration.